src/libproc_macro/lib.rs

```diff
@@ -66,6 +66,16 @@ pub struct LexError {
     _inner: (),
 }
 
+#[stable(feature = "proc_macro_lib", since = "1.15.0")]
+impl fmt::Display for LexError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.write_str("lex error")
```

Comment:
```suggestion
        f.write_str("cannot break string into tokens or parse it into a token stream")
```

Looking at the other `Display` impls for errors in the standard library, it seems that their wording isn't supposed to be stable, so it usually contains more details about the error.

Reply:
Oh yeah, that makes sense. The way I wrote it originally was mostly just concerned with putting *something* there.